ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -440,17 +462,6 @@ function check_upgrade_status() {
     echo -e "Upgrade checking start at $(date "+%F %T")\n"
     start_time=$(date "+%s")
 
-    # https://issues.redhat.com//browse/OTA-861
-    # When upgrade is processing, Upgradeable will be set to false
-    sleep 60 # while waiting for condition to populate
-    local case_id="OCP-25473"
```

Comment:
How about moving the block just behind https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh#L476-L481?

Reply:
this is in the while loop, if we put the check point into it will cause many time unnecessary execution.
but we can move below block out of  while loop to https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh#L394
```
        if [[ ${avail} == "True" && ${progress} == "False" && ${out} == *"Cluster version is ${cluster_version}" ]]; then
            echo -e "Upgrade checking end at $(date "+%F %T") - succeed\n"
            end_time=$(date "+%s")
            echo -e "Eclipsed Time: $(( ($end_time - $start_time) / 60 ))m\n"
            return 0
        fi
```

Reply:
done